.github/container/Dockerfile.base

```diff
@@ -5,13 +5,15 @@ ARG GIT_USER_EMAIL=jax@nvidia.com
 ARG SRC_MANIFEST_FILE=manifest.yaml
 ARG DEST_MANIFEST_DIR=/opt/manifest.d
 ARG CLANG_VERSION=17
+ARG NCCL_VERSION=""
 
 FROM ${BASE_IMAGE}
 ARG GIT_USER_EMAIL
 ARG GIT_USER_NAME
 ARG SRC_MANIFEST_FILE
 ARG DEST_MANIFEST_DIR
 ARG CLANG_VERSION
+ARG NCCL_VERSION
```

Comment:
We can also override the value of the `NCCL_VERSION` environment variable, which is set in the base image, to avoid confusion.